### PR TITLE
wallet: provide more detailed errors on failed NewWalletFromFile

### LINKED ISSUE #3180: can't initialize StateRoot service: open : no such file or directory
bad error description:

```
./neo-go-linux-amd64 node --config-file config.yml
...
can't initialize StateRoot service: open : no such file or directory
```

config.yml:
```
  StateRoot:
    Enabled: true
```

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Close #3180.

### CODE REVIEW

**Comment by AnnaShaleva** [comment]
@532910, is the following error OK for you?
```
can't initialize Oracle module: failed to open wallet file: open bad_orc_wallet.json: no such file or directory
```

File: pkg/wallet/wallet.go [hunk 1]
```diff
@@ -56,15 +57,15 @@ func NewWallet(location string) (*Wallet, error) {
 func NewWalletFromFile(path string) (*Wallet, error) {
 	file, err := os.Open(path)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("failed to open wallet: %w", err)
```
Comment (roman-khimov) [inline pkg/wallet/wallet.go:60]: `s/failed to//` ? Do we have `path` here? Can it be bad (requiring `%q`)?

**Comment by 532910** [comment]
remove second `open`:
```
can't initialize Oracle module: failed to open wallet file: bad_orc_wallet.json: no such file or directory
```

File: pkg/wallet/wallet.go (hunk 1 above)
Comment (AnnaShaleva) [reply pkg/wallet/wallet.go:60]: Fixed.

**Review by roman-khimov — APPROVED** [review]

File: pkg/wallet/wallet.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/wallet/wallet.go]: I see path is coming from https://pkg.go.dev/io/fs#PathError.Error in the resulting error string.

**Pull request merged**